Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,15 @@
-﻿using System;
+﻿using BootCamp.Chapter.Demos;
+using System;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-
```

Comment:
These are really lazy demos to be honest :D

Instead of spending time on creating an interactive menu like we did in previous exercises, I tried focusing on using high level language features such as generics, contravariant generic interfaces, and other sugar.